agent/src/heapstats-engines/jvmInfo.cpp

```diff
@@ -162,6 +162,7 @@ bool TJvmInfo::setHSVersion(jvmtiEnv *jvmti) {
       if (unlikely(result != 4)) {
         /*
          * Support JDK 9 EA
+         * See https://bugs.openjdk.java.net/browse/JDK-8061493
```

Comment:
If this PR relates to [this card](https://github.com/HeapStats/heapstats/projects/1#card-2231685) , please move it to "In Progress" or convert it to the issue.